internal/handlers/lokistack_create_or_update.go

```diff
@@ -102,12 +110,68 @@ func CreateOrUpdateLokiStack(ctx context.Context, req ctrl.Request, k k8s.Client
 			errCount++
 			continue
 		}
+
 		l.Info(fmt.Sprintf("Resource has been %s", op))
 	}
 
 	if errCount > 0 {
 		return kverrors.New("failed to configure lokistack resources", "name", req.NamespacedName)
 	}
 
+	// 1x.extra-small is used only for development, so the metrics will not
+	// be collected.
+	if string(opts.Stack.Size) != string(lokiv1beta1.SizeOneXExtraSmall) {
```

Comment:
No need to cast to `string` here. Both sides are of the same type.

Reply:
Still a string cast here

Reply:
sorry, missed that one. should be good now.

Reply:
Nope still casting to strings

Reply:
weird. _should_ be fine now...